webpack/scenes/ContentViews/Details/Versions/__tests__/contentViewVersions.test.js

```diff
@@ -152,6 +152,8 @@ test('Can show additional content and link to list page', async () => {
       .toHaveAttribute('href', '/content_views/5#/versions/11/files/');
     expect(getByText('1 Deb packages').closest('a'))
       .toHaveAttribute('href', '/versions/11/debPackages');
+    expect(getByText('80 Python packages', { exact: false })).not.toBeFalsy();
```

Comment:
I added `{ exact: false }` because the exact matching wasn't working. I think there are hidden characters that aren't being matched partially due to the `\n`s.  I may change this to a regex to be more specific.